src/Model/ICalExport.java

```diff
@@ -32,36 +32,30 @@
 import biweekly.util.Duration;
 
 /**
- * @author amila
- *
+ * @author Amila Dias
+ * CEG 3120 - RaiderPlanner
  */
+
 public class ICalExport{
 	private Date eStart;
 	private String title;
-	private MultilineString description;
 	private int hours;
 	private int minutes;
-	private int counter;
 	
-	public void createExportEvent(Event event, int counter){
+	public void createExportEvent(Event event, int counter) {
 		seteStart(event.getDate());
 		setTitle(event.getName());
 		setDescription(event.getDetails());
-		this.counter = counter;
```

Comment:
I do not understand why this line was removed.  It seems like the value of counter is no longer saved.